lib/src/countly_flutter.dart

```diff
@@ -102,7 +102,9 @@ class Countly {
 
   static Map<String, String> messagingMode = Platform.isAndroid ? {'TEST': '2', 'PRODUCTION': '0'} : {'TEST': '1', 'PRODUCTION': '0', 'ADHOC': '2'};
 
-  static Map<String, String> deviceIDType = {'TemporaryDeviceID': 'TemporaryDeviceID'};
+  static const temporaryDeviceID = 'CLYTemporaryDeviceID';
+  @Deprecated('This variable is deprecated, please use "Countly.temporaryDeviceID" instead')
```

Comment:
Should not we let them use interface and configuration methods rather than a static parameter
enabledTemporaryDeviceIdMode

Reply:
this sounds nice actually